src/docfx/lib/HtmlUtility.cs

```diff
@@ -269,9 +269,9 @@ public static string CreateHtmlMetaTags(JObject metadata, ICollection<string> ht
             return result.ToString();
         }
 
-        public static HtmlNode AddLinkType(this HtmlNode html, string locale)
+        public static HtmlNode AddLinkType(this HtmlNode html, string locale, Action<string> addLinkAction = null)
         {
-            AddLinkType(html, "a", "href", locale);
+            AddLinkType(html, "a", "href", locale, addLinkAction);
```

Comment:
will this cause duplicated links?

Reply:
Yes, could do a fix for it, ignore `external` link-type during resolving link. But the inclusion of `external` links can not be fixed.